### ISSUE TITLE
aria-label is not rendered when the ARROW keys are used in browse mode in firefox or Internet Explorer

### ISSUE THREAD

**Issue by rnickelson** [nvaccess/nvda#6362]
Make an html file containing the following text 
<input type=text aria-label="first Name"</input>
Open the file in Firefox or internet Explorer and navigate with the ARROW keys notice that the aria-label First name is not announced.

**Comment by ehollig** [comment]
Duplicate of #5710 4773

**Comment by ehollig** [comment]
Duplicate of #4773
